### DISCUSSION TITLE
PNPM

### DISCUSSION THREAD

**Discussion by stephenjason89** [nuxt/bridge#483] [Q&A] [answered]
### Environment

------------------------------
- Operating System: `Darwin`
- Node Version:     `v18.7.0`
- Nuxt Version:     `2.15.8`
- Package Manager:  `pnpm@7.9.0`
- Builder:          `webpack`
- User Config:      `buildDir`, `srcDir`, `bridge`, `components`, `ssr`, `dev`, `alias`, `generate`, `head`, `typescript`, `publicRuntimeConfig`, `privateRuntimeConfig`, `css`, `loading`, `loadingIndicator`, `plugins`, `buildModules`, `modules`, `proxy`, `vuetify`, `build`, `serverHandlers`, `devServerHandlers`
- Runtime Modules:  `@nuxtjs/vuetify@1.12.3`, `@nuxtjs/proxy@2.1.0`, `nuxt-social-meta@1.0.0`
- Build Modules:    `()`, `nuxt-storm@1.1.3`, `nuxt-compress@5.0.0`, `@pinia/nuxt@0.3.1`, `@vueuse/nuxt@9.1.0`, `~/modules/apolloSSR`, `@nuxt/bridge@3.0.0-27672061.a645f7f`
------------------------------

### Reproduction

<img width="1690" alt="image" src="https://user-images.githubusercontent.com/3810177/184473387-91177c2c-5536-45b1-91ef-eb6fa6154995.png">

### Describe the bug

I tried changing yarn 2 berry to pnpm
It shows me the error above

What I did was
remove yarn.lock
remove .yarn folder
remove node_modules

add .npmrc
<img width="307" alt="image" src="https://user-images.githubusercontent.com/3810177/184473428-ec1f8114-a6e6-4831-998e-0217e3dc1e93.png">

then tried 
pnpm install

Please let me know how to properly migrate from yarn2 to pnpm

Thank you

**Comment by danielroe** [comment, marked as answer]
I wouldn't auto-install peer dependencies, but the rest looks fine. I am not aware of any issues using pnpm with bridge.

**Reply by stephenjason89** [reply]
@danielroe Tried it again without the auto-install peer and it worked fine.

Should this be normal?
<img width="1647" alt="image" src="https://user-images.githubusercontent.com/3810177/184478386-829cc453-f9d6-4715-a862-521af9fb7b46.png">

pnpm i @pinia/nuxt

A lot of conflicting peer dependencies warning.
How to fix this warnings?

But yeah, looks like everything is working.


Thank you
